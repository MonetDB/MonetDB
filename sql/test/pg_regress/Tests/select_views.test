query TTT rowsort
SELECT * FROM street
----

query TT rowsort
SELECT name, thepath FROM iexit ORDER BY 1, 2
----

query TITI rowsort
SELECT * FROM toyemp WHERE name = 'sharon'
----

