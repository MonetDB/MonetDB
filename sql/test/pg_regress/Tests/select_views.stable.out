stdout of test 'select_views` in directory 'sql/test/pg_regress` itself:


# 17:11:18 >  
# 17:11:18 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=38959" "--set" "mapi_usock=/var/tmp/mtest-1142/.s.monetdb.38959" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/ufs/dinther/INSTALL/var/MonetDB/mTests_sql_test_pg_regress" "--set" "mal_listing=0"
# 17:11:18 >  

# MonetDB 5 server v11.18.0
# This is an unreleased version
# Serving database 'mTests_sql_test_pg_regress', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 15.356 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://uwakai.da.cwi.nl:38959/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-1142/.s.monetdb.38959
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded


# 17:11:19 >  
# 17:11:19 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-1142" "--port=38959"
# 17:11:19 >  

#--
#-- SELECT_VIEWS
#-- test the views defined in CREATE_VIEWS
#--
#SELECT * FROM street;
% sys.street,	sys.street,	sys.street # table_name
% name,	thepath,	cname # name
% clob,	clob,	clob # type
% 0,	0,	0 # length
#SELECT name, thepath FROM iexit ORDER BY 1, 2;
% sys.iexit,	sys.iexit # table_name
% name,	thepath # name
% clob,	clob # type
% 0,	0 # length
#SELECT * FROM toyemp WHERE name = 'sharon';
% sys.toyemp,	sys.toyemp,	sys.toyemp,	sys.toyemp # table_name
% name,	age,	location,	annualsal # name
% clob,	int,	clob,	bigint # type
% 0,	1,	0,	1 # length

# 17:11:19 >  
# 17:11:19 >  "Done."
# 17:11:19 >  

